src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java

```diff
@@ -2412,6 +2412,7 @@ private void createAzureResourceGroup(
             azureClient.resourceGroups()
                     .define(resourceGroupName)
                     .withRegion(locationName)
+                    .withTag(Constants.AZURE_JENKINS_TAG_NAME, Constants.AZURE_JENKINS_TAG_VALUE)
```

Comment:
Can we specify the tag value with a suffix? Like the vm-agents name. If user uses this plugin and creates more than one pair of resources in his subscription. He can associate with them

Reply:
We can also bind template name with the storage account and cloud name with the resource group.